lessons/Algebra2/exploring-linear-relationships/langs/en-us/index.adoc

```diff
@@ -0,0 +1,225 @@
+
+When we see patterns in data, we can use those patterns to __make predictions__ based on that data. We can even draw a line to show all the possible predictions at once! These predictions represent our "best guess" at the underlying relationship in the data, as we try to model that relationship using math.
+
+These models are just functions being graphed on top of the scatter plot, with the goal of minimizing the distance between the line and all the points on the plot. For straight-line relationships, these are _linear functions_ or "linear models". You may also have heard of linear models referred to as the @vocab{line of best fit}.
+
+When we make a model, we want it to be the closest possible approximation of all the points. A "good fit" has most of the points very close to the line, and a "bad fit" has the points very far away.
+
+
+=== Investigate
+
+If you could draw linear model through the cloud of points in your scatter plot, where would you draw it? What would the slope and y-intercept of that model be?
+
+[.lesson-instruction]
+Complete @printable-exercise{model-college-v-income.adoc}
+
+The `fit model` function consumes columns for our xs and yes, __as well as a function__. It draws a scatter plot and then displays the function on top of it. The `fit-model` also calculates __how well the function models the data__.
+
+Have students discuss their answers:
+
+* What did you Notice and Wonder, the first time you ran `fit-model`?
+** (Answers will vary)
+* Did you notice something called @math{R^2} on the display? What number did it show?
+** -15.63
+* What linear model gave you the best fit? What was the @math{R^2}, slope and y-intercept?
+** (Answers will vary)
+* What do you think @math{R^2} tells us about a model?
+** It tells you how well the model fits the data. When @math{R^2} is higher, the model fits better.
+* How did you fill in the blanks for the last question on the page?
+** (Answers will vary)
+
+@math{R^2} describes the _percentage of the variation in the y-variable that is explained by the x variable_ in our model. In other words, an @math{R^2} value of 0.20 could mean that “20% of the variation in median household income is explained by the percentage of college degrees in a state, according to our linear model”. Better models will explain a higher percentage of that variation!
+
+If the model is a perfect, the math{R^2} value will be 1.00, meaning 100% of the @math{y}-values can be explained by the @math{x}-values. Of course in the real world, no model is perfect! The @math{R^2} value for no correlation at all is *zero*. If we just drew a horizontal predictor line in the middle of the points on the scatter plot, it would mean that we expect a median income somewhere in that range but with no connection whatsoever to the percentage of people who finish college.
+
+[.lesson-instruction]
+But what if we drew a line that wasn't even in the cloud at all? Try changing your predictor to do just that.
+
```

Comment:
If you revise the worksheet as I've suggested, this question could be omitted here, I think.